R/scripts.R

```diff
@@ -99,12 +106,31 @@ run_civis <- function(expr, ...) {
 #'
 #' # Run the template
 #' run_template(id, arguments = list(arg1 = 1, arg2 = 2), ...)
+#'
+#' # Run the template and return JSON value outputs
+#' run_template(id, arguments = list(arg1 = 1, arg2 = 2), JSONValue=TRUE...)
 #' }
-run_template <- function(id, arguments, ...) {
+run_template <- function(id, arguments, JSONValue=FALSE, ...) {
   job <- scripts_post_custom(id, arguments = arguments, ...)
   run <- scripts_post_custom_runs(job$id)
   await(scripts_get_custom_runs, id = job$id, run_id = run$id)
-  fetch_output_file_ids(civis_script(job$id, run$id))
+  if (JSONValue) {
+     output <- fetch_output(civis_script(job$id, run$id))
+     json_output <- output[sapply(output, function(o) o$objectType=="JSONValue")]
+     if (length(json_output) > 1) {
+        cat("Error in returning JSON outputs of template run -- too many JSON outputs")
```

Comment:
`warning` seems more appropriate than `cat` here.

Reply:
👍 changed to warning.